user/src/main/java/org/the_chance/honeymart/ui/feature/cart/composables/CartItem.kt

```diff
@@ -88,7 +88,7 @@ fun CartItem(
                 }
             )
             Text(
-                text = formatCurrencyWithNearestFraction(product.productPrice),
+                text = formatCurrencyWithNearestFraction(product.productPrice * product.productCount),
```

Comment:
I think this calculation shouldn't be done within the screen scope. instead, it should be provided as a state from the view model and assigned directly to this Text composable.

Reply:
the price was intended to give the price of one item not the total @IbrahimAlzaidi